### PR TITLE
fix handling of clicks on lightbox items

### PR DESCRIPTION
**Pull request by tosyu**
This fixes catching of click events globally, its a little hacky but I do not think refactoring to strip out jQuery is needed

Signed-off-by: Krzysztof Antoszek <krzysztof@antoszek.net>

### CODE REVIEW

File: public/js/lightbox.js [hunk 1]
```diff
@@ -47,11 +47,27 @@
     // Loop through anchors and areamaps looking for either data-lightbox attributes or rel attributes
     // that contain 'lightbox'. When these are clicked, start lightbox.
     Lightbox.prototype.enable = function() {
-      var self = this;
-      $('body').on('click', 'a[rel^=lightbox], area[rel^=lightbox], a[data-lightbox], area[data-lightbox]', function(event) {
-        self.start($(event.currentTarget));
-        return false;
-      });
+      var self = this,
+				validSelectors = 'a[rel^=lightbox], area[rel^=lightbox], a[data-lightbox], area[data-lightbox]';
```
Comment (mateusz-lisik) [inline public/js/lightbox.js]: Personally I'd put valid selectors as variable with separate var keyword. AFAIR with ES6 such thing will become obsolete.

**Review by mateusz-lisik — CHANGES_REQUESTED** [review]
Seems good for me, but I would fix indentation of the code sice rest of the file is properly formatted.

File: public/js/lightbox.js (hunk 1 above)
Comment (tosyu) [reply public/js/lightbox.js]: habits ;-)

**Comment by tosyu** [comment]
@matteprl updated

**Review by mateusz-lisik — APPROVED** [review]
Perfect

**Pull request merged**